pkgs/development/embedded/arduino/arduino-cli/default.nix

```diff
@@ -46,7 +59,9 @@ if stdenv.isLinux then
 
     extraInstallCommands = ''
       mv $out/bin/$name $out/bin/arduino-cli
+      cp -r ${pkg.outPath}/share $out/share
     '';
+    passthru.pureGoPkg = pkg;
```

Comment:
```suggestion
    passthru.pureGoPkg = pkg;
```
Why is it here?

Reply:
> Why is it here?

I think it's useful to be able to inspect this package when one wishes to make changes to the Fhs wrapping package. It should also be slightly easier to override it when this passthru is available. To make it completely easy to override the wrapped and the wraping packages, further changes are needed, but I was thinking this is a good start..

Perhaps the maintainer would like to see a separate commit / separate PR that will enable full overriding of the package, and I'll remove this line for this commit..

Reply:
I was just trying to understand. I have no opinion on this.